dotnet/src/Connectors/Connectors.OpenAIV2/Core/ClientCore.cs

```diff
@@ -28,6 +34,16 @@ namespace Microsoft.SemanticKernel.Connectors.OpenAI;
 /// </summary>
 internal partial class ClientCore
 {
+    /// <summary>
+    /// White space constant.
+    /// </summary>
+    private const string WhiteSpace = " ";
```

Comment:
What about `SingleSpace` as the name as `WhiteSpace` could be lot's of different characters